content/content-tool/tool/src/webapp/vm/resources/sakai_create_uploads.vm

```diff
@@ -216,9 +253,25 @@ Dropzone.options.fileUploader = {
             file.previewTemplate.appendChild(document.createTextNode(responseText));
         });
 
+        // Client side copyright check
         submitButton.addEventListener("click", function() {
-            submitButton.disabled=true;
-            dz.processQueue(); // Tell Dropzone to process all queued files.
+            var requireChoice = $copyright_requireChoice;
+            var copyrightSelect = document.getElementById('copyright');
+
+            // 'Please select...' is always the first element in the drop down, but only enforce this if configured via sakai.properties
+            if (requireChoice && copyrightSelect !== null && copyrightSelect.selectedIndex < 1)
```

Comment:
is this chunk of script code duplication?

Reply:
Yes, but pieces of the `else` statement are unique. I'll extract the common parts to a function in `sakai_properties.vm` (which both `sakai_resources_properties.vm` and `sakai_create_uploads.vm` consume).